tools/xtu-build-new/xtu-analyze.py

```diff
@@ -195,6 +203,15 @@ def check_executable_available(exe_name, arg_path):
 dircmd_2_original_orders = {}
 for step in buildlog:
     if src_pattern.match(step['file']):
+        if 'command' not in step:
+            if 'arguments' in step:
+                # change arguments to command
+                command_from_arg = get_command_from_arguments(step['arguments'])
+                step['command'] = command_from_arg
+            else:
+                print "have no field \"command\" and \"arguments\" to deal, exit..."
```

Comment:
Prefer `print(...)` which is compatible with python 3.

Reply:
I see what you mean, let me make a change :)